### PR TITLE
Journal week 4

### PR DESCRIPTION
**Pull request by ericquayson**

### CODE REVIEW

**Comment by jmcreasman** [comment]
Love your idea for a tutorial!

**Comment by jeremyrist** [comment]
:+1: Would like to see that tutorial as well!

File: journal-week-4.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+## Outline
+
+### Pick 5 things to talk about
+
++ MVC and Chessboard
++ Starting to get more comfortable with CSS and SASS 
++ Talk about the trouble I had with compiling the files with node-sass
++ Talk about feeling more confident in my skills as the program contiinues
```
Comment (kellymurray) [inline journal-week-4.md]: :+1:

**Comment by jeremyrist** [comment]
:+1: Good job, the @media queries is something I'm not so confident with _yet_. Thanks

**Comment by Ricecoder** [comment]
Good job with taking CSS head on! Also thanks for the tutorial on media queries. I will definitely be referencing it in the future. :+1:

**Pull request merged**